ui-tests/src/test/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java

```diff
@@ -78,6 +78,8 @@
     private static final By ACTIVE_VISIT_MESSAGE = By.cssSelector("active-visit-message");
     private static final By ALLERGIES_LINK = By.id("allergyui-editAllergies");
     private static final By CONDITIONS_LINK = By.cssSelector(".conditions .info-header i.right");
+    private static final By ATTACHMENTS_LINK = By.cssSelector("a[id='attachments.attachments.visitActions.default']");
+    private static final By ACTIVE_VISIT = By.cssSelector("a.ng-binding");
```

Comment:
this css needs to be more improved, work with better element lookup, consider parent elements to restrict what can be retrieved here